### ISSUE TITLE
Error Bounce: error scanning bounce mailbox: multipart: NextPart: EOF

### ISSUE THREAD

**Issue by anabeatrizmr** [knadh/listmonk#1848]
**Version:**
 - listmonk: [2.5 0]

Our problem is that the bounce isn't working, it shows the following error message: error scanning bounce mailbox: multipart: NextPart: EOF

we've tryied a lot of configurations, but still not working, can u guys please help? i didn't found anything about this error in internet

**Comment by AnonWG4** [comment]
🎫 SMTP, Scam Pages, Links, Hosting, Sender, Leads, Extractor, Verifier

🎟Email Spamming Tools 100% Inbox
🎟Sms Spamming Tools 100% Inbox
🎟Office365 Spammimg Tools 100% inbox
🎟Banks Spammimg Tools 100% inbox

✅Scam Pages, True Login & Anti Bot, uAdmin
🍞Bank Scam Pages
🍞Greece/USA/UK/French Scam Pages
🍞Crypto Scam Pages
🍞Office365 Cookies Grabber

✅Bullet proof Link+SSL+Hosting

✅Email Senders Crack/Encrypted
🍞All kind of sender available

✅SMTP To Sms Sender
🍞Smtp to sms php sender
🍞Sender Id
🍞Php Sms Sender USA

✅ Encrypted Letter To Inbox
🍞 Show sample and get inbox guaranteed letter.

✅Email Leads Extractor Any Country
✅Email Leads Verifier
✅Sms Sender php Version
✅Sms Leads Extractor

✅SMTP Servers

🔥24/7 Customer Support
🔥Telegram, Channel
🔥All Proof on Fire 🔥
🔥Dont wait, work with me now and become a permanent Customer

✅ Telegram : @AnonWG4
✅ Channel Link : https://t.me/+Z4BjtNeklvNjYmI0

**Comment by sspaeti** [comment]
same here

**Comment by Savanta** [comment]
Same here

**Comment by Aurely9n** [comment]
Same here...

**Comment by trailfog** [comment]
Same issue here, using v6.1.0. Had a 30GB log file (roughly 6'000'000 log entries per second) filled with `pop.go:133: error reading multipart bounce message [message-number]: multipart: NextPart: EOF` 

After checking the mail-inbox, it was caused by a message that had no body.

**Comment by sspaeti** [comment]
@trailfog How did you find empty messages? I tried to delete the smallest email that might have been missing the body. But I still get 1000s of logs every second. I tried to break the loop with a `break` (instead of `continue`), but it didn't really solve it: https://github.com/sspaeti/listmonk-code/commit/70af77dbaba8a3b5f1cc6f5dc16215d3c9b44649

**Update - Never mind** - I just set up the AWS endpoint (Perfectly described in the docs at https://listmonk.app/docs/bounces/#amazon-simple-email-service-ses) and do not need the Pop email, which I set up to my actual email, where I get bounced notifications, but also my regular email, so that's why I got these error messages. So all good in my case, just a wrong configuration.
